src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -956,6 +956,26 @@ They are available in two places: `beforeSend`/`beforeBreadcrumb` and `eventProc
 
 #### Hints for Events
 
+##### Use-Case: `before-send`
+
+The `before-send` callback is passed the event and a second argument `hint` which holds one or more hints. Typically this hint holds the original exception so that additional data can be extracted or grouping is affected. `before-send` will be called right before the event is sent to the server, so it’s the last place where you can edit its data.
```

Comment:
> so that additional data can be extracted or grouping is affected
Not sure what you mean by "grouping is affected" here, but IMHO you get enough meaning by just leaving it off.